src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +68,60 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={query}
+              onChange={(event) => {
+                setQuery(event.target.value);
+                setError(false);
+              }}
             />
           </div>
+          {error && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              className={classNames('button is-light', {
+                'is-loading': loading,
+              })}
+              disabled={!query}
             >
               Find a movie
             </button>
           </div>
+          {movie && (
+            <div className="control">
+              <button
+                data-cy="addButton"
+                type="button"
+                className="button is-primary"
+                onClick={() => {
+                  addMovie(movie);
+                  setQuery('');
+                  setError(false);
```

Comment:
is any chance that error is `true` if the movie was fetched?

Reply:
maybe a little) I deleted setError